### PR TITLE
we need more php love and understanding

### PR DESCRIPTION
**Pull request by fsniper**

### CODE REVIEW

**Comment by ProgVal** [comment]
You should consider implementing [PSR 8](https://github.com/php-fig/fig-standards/blob/master/proposed/psr-8-hug/psr-8-hug.md)

**Comment by fsniper** [comment]
Oh Huggable Objects! I will do that as a next step and a new pr. Every user should be PSR 8 compatible.

File: php_love.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+echo "No php love? Don't be so snob!";
+
+$users = [
+  "I", "You, ""he", "she", "it",
+  "We", "They"
+];
+
+foreach ($users as $user) {
+  echo "A user: $user";
```
Comment (mrkrstphr) [inline php_love.php]: No newline?

File: php_love.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+}
+
+?>
```
Comment (mrkrstphr) [inline php_love.php]: Use of end tag is discouraged

**Comment by mrkrstphr** [comment]
Typical PHP...

**Pull request merged**